core/src/html/layout.rs

```diff
@@ -491,8 +491,9 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
             "_serif" => Some(DefaultFont::Serif),
             "_sans" => Some(DefaultFont::Sans),
             "_typewriter" => Some(DefaultFont::Typewriter),
-            // [NA] I suspect that there might be undocumented more aliases.
-            // I think I've seen translated versions of these used in the wild...
+            "_ゴシック" => Some(DefaultFont::JapaneseGothic),
+            "_等幅" => Some(DefaultFont::JapaneseGothicMono),
+            "_明朝" => Some(DefaultFont::JapaneseMincho),
             _ => None,
         } {
             return context
```

Comment:
I can't link it? But on line 525, I wrote "one of the three default" (fonts), we should just remove "three".